server.go

```diff
@@ -3065,6 +3065,7 @@ func createChannelUpdate(info *channeldb.ChannelEdgeInfo,
 		ChannelFlags:    policy.ChannelFlags,
 		TimeLockDelta:   policy.TimeLockDelta,
 		HtlcMinimumMsat: policy.MinHTLC,
+		HtlcMaximumMsat: policy.MaxHTLC,
```

Comment:
Commit message nor body doesn't match this diff. For ref it's: 
> server: set max htlc in ChannelUpdate sent on HtlcSatifiesPolicy failure

and this diff touches only the server.

Reply:
I think maybe it's because `HtlcSatifiesPolicy` eventually calls this method when fetching the updated `ChannelUpdate`? @valentinewallace

Reply:
@halseth that's what I was originally thinking, but checking again now it looks like I may be mistaken. This function gets called down the stack by `announceChanStatus`, which is used in 4+ places for different reasons.